package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -628,6 +628,30 @@ def __init__(self, filename, n_atoms=None, mmap=None, **kwargs):
         # load first data frame
         self._read_frame(0)
 
+    def __getstate__(self):
+        state = self.__dict__.copy()
+        del state['_f']
+
+        return state
+
+    def __setstate__(self, state):
+        self.__dict__.update(state)
+        self._f = scipy.io.netcdf.netcdf_file(self.filename,
+                                              mmap=self._mmap)
+
+#    @property
```

Comment:
Sorry just reading through the PR in chunks at a time but as an initial comment, any reason for this commented code?

Reply:
Looks like it was code that was added in @richardjgowers's original PR in conjunction with removing the code for setting `self.n_frames` in `__init__`, which has since been altered and so still in this PR up [here](https://github.com/MDAnalysis/mdanalysis/blob/e9121df8f7ab57ed0de68d48a1e7fdd8e3afb466/package/MDAnalysis/coordinates/TRJ.py#L552) . I'm not sure what the original intention for the move was, though, so don't know if it's better to remove this or move the updated `n_frames` bit here